### ISSUE TITLE
Enable flaky test "Disable_Writes_To_DisableContainerFile_Restarts_ScriptHost"

### ISSUE THREAD

**Issue by pragnagopa** [Azure/azure-functions-host#6994]
Test - Disable_Writes_To_DisableContainerFile_Restarts_ScriptHost is faling frequently in V2 PR builds. Disabled the test to unblock release.

@balag0 - can you take a look?

**Comment by balag0** [comment]
Thanks. This api has been replaced by drain mode api and not used anymore. Will remove the test and api. Okay to ignore the test for now.

**Comment by satvu** [comment]
Closing as complete / no longer relevant.
